packages/config/src/api/ConfigProfiles.ts

```diff
@@ -110,6 +110,30 @@ export class ConfigProfiles extends ConfigApi {
         return shortPath.replace(/(^|\.)/g, "$1profiles.");
     }
 
+    // _______________________________________________________________________
+    /**
+     * Obtain the profile name (either nested or not) based on a property path.
+     *
+     * @param path The property path.
+     *
+     * @returns The corresponding profile name.
+     *
+     * @note This may be useful for supporting token authentication in a nested configuration
+     *
+     */
+    public getProfileNameFromPath(path: string): string {
```

Comment:
Do you think we should deprecate `expandPath` and rename it to `getProfilePathFromName` to make both method names consistent?

Reply:
or should we name this one `shrinkPath`? 😋